plugins/generator-1.15.2/forge-1.15.2/templates/plant.java.ftl

```diff
@@ -375,9 +375,20 @@ import net.minecraft.block.material.Material;
         </#if>
 
 		<#if (data.canBePlacedOn?size > 0)>
-		@Override public boolean isValidGround(BlockState state, IBlockReader worldIn, BlockPos pos) {
+		<#if data.plantType != "growapable">@Override</#if>

```

Comment:
Why is only @override part in if? Override by itself does not do any sematic change to the code.

Reply:
Because the code must be there even if the plant is "growapable" type. The only difference is that it does not override anything if it's "growapable".

Reply:
Understood